app/src/main/java/org/fossasia/openevent/general/search/SearchTimeActivity.kt

```diff
@@ -25,19 +30,64 @@ class SearchTimeActivity : AppCompatActivity() {
         this.supportActionBar?.title = ""
 
         val calendar = Calendar.getInstance()
+        val dateFormat = "yyyy-MM-dd"
+        val simpleDateFormat = SimpleDateFormat(dateFormat, Locale.US)
```

Comment:
cant the same result be acheived using date formatting functions from event utils?

Reply:
@simarsingh24  currently I don't find any suitable should I create one or currrent way is ok?

Reply:
in that case IMO add a function in eventUtils class. Best would be to wait for @iamareebjamal comment

Reply:
added for now @iamareebjamal  pls review